rarible/src/modules/abi.ts

```diff
@@ -0,0 +1,358 @@
+import { Address, BigInt, ByteArray, Bytes, ethereum } from "@graphprotocol/graph-ts";
+import { log } from "matchstick-as";
+
+export namespace abi {
```

Comment:
It seems, this file has code specific to seaport subgraph. Could you please delete functions which are not used.